cmd/yurthub/app/options/options.go

```diff
@@ -72,6 +72,8 @@ type YurtHubOptions struct {
 	EnableIptables            bool
 	HubAgentDummyIfIP         string
 	HubAgentDummyIfName       string
+	CloudProviderIP           string
+	APIServerIPs              []string
```

Comment:
I think we don't need to add `APIServerIPs` filed, because there is `ServerAddr` field that used for specifying the address of cluster control-plane.

Reply:
done